resources/ts/Components/ActivationDialog.tsx

```diff
@@ -11,6 +11,59 @@ import { useIsMobile } from "../hooks/useIsMobile";
 import { Subscription } from "../types/Subscription";
 import { PricingPlan } from "../hooks/useGetPricingPlans";
 import { formatCurrency } from "../utils/formatCurrency";
+import { Heading } from "./Heading";
+import { CouponInfoContext } from "../Context/CouponInfo";
+
+const PriceOverview = ({
+  coupon,
+  subscription,
+  pricingPlan,
+}: {
+  coupon?: string;
+  subscription: Subscription;
+  pricingPlan: PricingPlan;
+}) => {
+  const { t } = useTranslation();
+
+  const couponInfo = useContext(CouponInfoContext);
+
+  return (
+    <section className="flex flex-col flex-1 gap-2 border-t mt-4 pt-4 text-sm">
+      <Heading level={4}>
+        {t("integrations.activation_dialog.price_overview.title")}
+      </Heading>
+
+      <div className="grid grid-cols-2 gap-1">
+        <span>{`${pricingPlan.title} ${t("integrations.activation_dialog.price_overview.setup")}`}</span>
+        <span>{formatCurrency(subscription.currency, subscription.fee)}</span>
+
+        {!!coupon && (
+          <>
+            <span className="text-publiq-orange">
+              {t("integrations.activation_dialog.price_overview.coupon")}
+            </span>
+            <span className="text-publiq-orange">
+              {`- ${formatCurrency(
+                subscription.currency,
+                couponInfo.reductionAmount
+              )}`}
+            </span>
+          </>
+        )}
+
+        <span>{`${pricingPlan.title} ${t("integrations.activation_dialog.price_overview.plan")}`}</span>
```

Comment:
Is there a reason why you need template literals here?
```suggestion
        <span>{pricingPlan.title} {t("integrations.activation_dialog.price_overview.plan")}</span>
```